processor/src/main/java/com/linecorp/decaton/processor/metrics/Metrics.java

```diff
@@ -99,6 +102,23 @@ public void close() {
     }
 
     public class SubscriptionMetrics extends AbstractMetrics {
+        volatile KafkaClientMetrics kafkaClientMetrics;
+
+        public void bindClientMetrics(Consumer<String, byte[]> consumer) {
+            if (kafkaClientMetrics != null) {
+                closeClientMetrics();
```

Comment:
In which case would this be effective?

Reply:
Ah, none. Indeed `SubscriptionMetrics` are only instantiated in `ProcessorSubscription` ctor. Removing